src/handler.rs

```diff
@@ -8,49 +8,49 @@ use arbiter::Arbiter;
 use context::Context;
 use fut::{self, ActorFuture};
 
-/// Message handler
+/// A trait for objects which can handle messages of a specific type.
 ///
-/// `Handler` implementation is a general way how to handle
-/// incoming messages, streams, futures.
+/// Implementing `Handler` is a general way to handle incoming
+/// messages, streams, and futures.
 ///
-/// `M` is a message which can be handled by the actor.
+/// The type `M` is a message which can be handled by the actor.
 #[allow(unused_variables)]
 pub trait Handler<M>
 where
     Self: Actor,
     M: Message,
 {
-    /// The type of value that this handle will return
+    /// The type of value that this handler will return.
     type Result: MessageResponse<Self, M>;
 
-    /// Method is called for every message received by this Actor
+    /// This method is called for every message received by this actor.
     fn handle(&mut self, msg: M, ctx: &mut Self::Context) -> Self::Result;
 }
 
-/// Message type
+/// A trait for objects that represent messages.
```

Comment:
A trait describing message that can be handled by `Actor`

Reply:
I've left the "object" in there, but have extended the description a bit, as suggested.